frontend/src/metabase/entities/containers/EntityName.tsx

```diff
@@ -0,0 +1,53 @@
+import type Question from "metabase-lib/Question";
+
+import EntityObjectLoader from "./EntityObjectLoader";
+
+type EntityId = string | number;
+
+interface EntityNameProps {
+  entityType: string;
+  entityId: EntityId;
+  property?: string;
+}
+
+interface EntityWrapper {
+  getName: () => string;
+}
+
+export const EntityName = ({
+  entityType,
+  entityId,
+  property = "name",
+}: EntityNameProps) => {
+  // This is a special case for questions, because we're returning `metabase-lib/Question`
+  // from question entity's `getObject` in https://github.com/metabase/metabase/pull/30729.
+  // If we wrap it in `EntityWrapper`, we'd lose all properties from `metabase-lib/Question`.
+  if (entityType === "questions") {
```

Comment:
The difference, in this case, is not passing `wrapped` i.e. not wrapping `metabase-lib/Question` object inside an `EntityWrapper`. This way we could call `question.displayName()` which will return the question name. Had we pass `wrapped` like other entities, we'd lose all functions from `metabase-lib/Question` and there wouldn be no way to access the question name unless we directly access `question._card.name` directly, which looks too hacky.